### PR TITLE
Initial commit for this recipe

### PR DESCRIPTION
**Pull request by mhearne-usgs**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mhearne-usgs** [comment]
@conda-forge/help-python Can someone please merge this? Thank you in advance!

**Comment by mhearne-usgs** [comment]
@ocefpaf I requested a review from the python team - is there someone else I should be pinging?

File: recipes/esi-utils-cartopy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/esi-utils-cartopy/meta.yaml:1]: Ping!

File: recipes/esi-utils-cartopy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - cartopy
+    - matplotlib-base
+    - numpy
+    - pyproj
```
Comment (ocefpaf) [inline recipes/esi-utils-cartopy/meta.yaml:54]: Upstream declares only `esi_utils_geo>=0.9.0`.

File: recipes/esi-utils-cartopy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    - esi-utils-geo
```
Comment (ocefpaf) [inline recipes/esi-utils-cartopy/meta.yaml:55]:
```suggestion
    - esi-utils-geo >=0.9.0
```

File: recipes/esi-utils-cartopy/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  # More info about selectors can be found in the conda-build docs: 
+  # https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
```
Comment (ocefpaf) [inline recipes/esi-utils-cartopy/meta.yaml:45]: This one does not seem to have any compiled extension.

```suggestion
```

File: recipes/esi-utils-cartopy/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
```
Comment (ocefpaf) [inline recipes/esi-utils-cartopy/meta.yaml:32]:
```suggestion
  noarch: python
```

File: recipes/esi-utils-cartopy/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+  host:
+    - python
```
Comment (ocefpaf) [inline recipes/esi-utils-cartopy/meta.yaml:47]:
```suggestion
    - python >=3.8
```

File: recipes/esi-utils-cartopy/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+    - pip
+  run:
+    - python
```
Comment (ocefpaf) [inline recipes/esi-utils-cartopy/meta.yaml:50]:
```suggestion
    - python >=3.8
```

**Review by ocefpaf — CHANGES_REQUESTED** [review]

**Comment by mhearne-usgs** [comment]
@conda-forge-admin please rerender

**Comment by mhearne-usgs** [comment]
@ocefpaf The linux docker build failed, but it *looks* like its a network error. Tried re-rendering to restart the job, but that didn't appear to do what I wanted. If in fact this is a network error and I need to re-run the linux pipeline, how do I accomplish that?

**Comment by ocefpaf** [comment]
> Tried re-rendering to restart the job, but that didn't appear to do what I wanted

re-render only works for feedstocks

> how do I accomplish that?

Closing and re-opening works. Or send a new commit. You still need to remove all the comments and to fix the license? I
I'll send those commits in a moment.

**Pull request merged**